app/templates/components/list.hbs

```diff
@@ -50,7 +49,7 @@ as |content|
       maxWidth=column.maxWidth
       on-drag=(action "didResize" column.id)
       position=(one-way column.width)
-      setIsDragging=setIsDragging
+      setIsDragging=@setIsDragging
```

Comment:
It seems like some of the test failures point to `@setIsDragging`. Should it remain as `setIsDragging`?

Reply:
I think it's that this stuff is incompatible with older Ember versions, not sure how to best handle this

Reply:
We need to add ember-named-arguments-polyfill I believe...

Reply:
gah, looks like https://github.com/rwjblue/ember-named-arguments-polyfill only goes down to Ember 2.10 😩

Reply:
@rwjblue so adding that polyfill gets things passing on 2.10+. It seems like versions before that still have the error. Thoughts?

Reply:
Heh, I needed to refresh my page, you noticed the same things first 😛